charts/pgadmin4/templates/deployment.yaml

```diff
@@ -121,8 +121,7 @@ spec:
               value: /pgadmin4/servers.json
           {{- end }}
           {{- range .Values.env.variables }}
-            - name: {{ .name | quote }}
-              value: {{ .value | quote }}
+            - {{ . | toJson }}
```

Comment:
```suggestion
            {{ toYaml .Values.env.variables | indent 12 }}
```

Reply:
i dont think you can do toYaml, it errors out because it keeps getting confused on the indentation

Reply:
I tried it and you get the error message "Error: YAML parse error on pgadmin4/templates/deployment.yaml: error converting YAML to JSON: yaml: line 61: mapping values are not allowed in this context"

Reply:
ah my bad, you can change that part toJson - and it should work.

Reply:
i think it needs to be how it currently is, in the pr, because you want to keep the array definition as yaml, just the array contents to be json

Reply:
does it still needs to be a range though?